assets/js/modules/reader-revenue-manager/components/dashboard/RRMIntroductoryOverlayNotification.js

```diff
@@ -0,0 +1,191 @@
+import useDashboardType from '../../../../hooks/useDashboardType';
+import ExternalIcon from '../../../../../svg/icons/external.svg';
+import { trackEvent } from '../../../../util';
+import { Button } from 'googlesitekit-components';
+import { useSelect, useDispatch } from 'googlesitekit-data';
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { CORE_UI } from '../../../../googlesitekit/datastore/ui/constants';
+import { VIEW_CONTEXT_MAIN_DASHBOARD } from '../../../../googlesitekit/constants';
+import {
+	MODULES_READER_REVENUE_MANAGER,
+	READER_REVENUE_MANAGER_MODULE_SLUG,
+	PUBLICATION_ONBOARDING_STATES,
+} from '../../datastore/constants';
+import whenActive from '../../../../util/when-active';
+import { createInterpolateElement } from '@wordpress/element';
+import SupportLink from '../../../../components/SupportLink';
+
+const { ONBOARDING_COMPLETE } = PUBLICATION_ONBOARDING_STATES;
+
+export const RRM_MONETIZATION_OVERLAY_NOTIFICATION =
+	'rrmMonetizationOverlayNotification';
+
+function RRMIntroductoryOverlayNotification() {
+	const viewContext = useViewContext();
+	const isViewOnly = useViewOnly();
+	const dashboardType = useDashboardType();
+
+	const { publicationID, publicationOnboardingState, paymentOption } =
+		useSelect(
+			( select ) =>
+				select( MODULES_READER_REVENUE_MANAGER ).getSettings() || {}
+		);
+
+	const isDismissed = useSelect( ( select ) =>
+		select( CORE_USER ).isItemDismissed(
+			RRM_MONETIZATION_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const { dismissOverlayNotification } = useDispatch( CORE_UI );
+
+	const usingNonMonetaryCTA =
+		paymentOption === '' || paymentOption === 'noPayment';
+
+	const shouldShowNotification =
+		isDismissed === false &&
+		! isViewOnly &&
+		dashboardType === VIEW_CONTEXT_MAIN_DASHBOARD &&
+		publicationOnboardingState === ONBOARDING_COMPLETE &&
+		usingNonMonetaryCTA;
+
+	const isDismissing = useSelect( ( select ) =>
+		select( CORE_USER ).isDismissingItem(
+			RRM_MONETIZATION_OVERLAY_NOTIFICATION
+		)
+	);
+
+	const serviceURL = useSelect( ( select ) =>
+		select( MODULES_READER_REVENUE_MANAGER ).getServiceURL( {
+			path: 'reader-revenue-manager',
+			query: {
+				publication: publicationID,
+			},
+		} )
+	);
+
+	const dismissNotice = () => {
+		dismissOverlayNotification( RRM_MONETIZATION_OVERLAY_NOTIFICATION );
+	};
+
+	const dismissNotification = () => {
+		trackEvent(
+			`${ viewContext }_rrm-monetization-notification`,
+			'dismiss_notification'
+		).finally( () => {
+			dismissNotice();
+		} );
+	};
+
+	return (
+		<OverlayNotification
+			className="googlesitekit-reader-revenue-manager-overlay-notification googlesitekit-reader-revenue-manager-monetization-notification"
+			GraphicDesktop={ ReaderRevenueManagerIntroductoryGraphicDesktop }
+			GraphicMobile={ ReaderRevenueManagerIntroductoryGraphicMobile }
+			onShow={ () => {
+				trackEvent(
+					`${ viewContext }_rrm-monetization-notification`,
+					'view_notification'
+				);
+			} }
```

Comment:
GA events have not yet been defined for these areas, and it is likely that there will be additional labels for this component. Let's not add these trackers now.